.github/workflows/image-arm.yaml

```diff
@@ -116,3 +116,40 @@ jobs:
           name: ${{ matrix.flavor }}-image
           path: build
           if-no-files-found: error
+  iso:
+    runs-on: ubuntu-latest
+    needs:
+      - get-matrix
+    strategy:
+      fail-fast: false
+      matrix:
+        flavor:
+          - "opensuse-leap"
+    steps:
+      - uses: actions/checkout@v3
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+      - name: Install earthly
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        uses: Luet-lab/luet-install-action@v1
+        with:
+          repository: quay.io/kairos/packages
+          packages: utils/earthly
+      - name: Set up QEMU
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: all
+      - name: Set up Docker Buildx
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build iso  🔧
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        run: earthly --platform=linux/arm64 +iso --FLAVOR=${{ matrix.flavor }}
+      - uses: actions/upload-artifact@v3
```

Comment:
I guess we need to adapt as well the release pipeline?

Reply:
I haven't touched release pipelines as I wanted to discuss on Monday what we need to support and how regarding arm64. Flavours, best-effort basis for arm64 issues, etc...